tools/test-cli-fatjar/src/main/java/com/microsoft/playwright/testclifatjar/TestApp.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.playwright.testclifatjar;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.impl.driver.Driver;
+import com.microsoft.playwright.impl.driver.jar.DriverJar;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.util.Collections;
+
+public class TestApp {
+  public static void main(String[] args) throws IOException, URISyntaxException {
+    DriverJar driver = (DriverJar) Driver.ensureDriverInstalled(Collections.emptyMap(), true);
```

Comment:
If you do this, subsequent call Playwright.create() below will not actually trigger  DriverJar.initFileSystem second time (when a filesystem already exists). Just make DriverJar.getDriverResourceURI a static method and call it here without calling Driver.ensureDriverInstalled, it should work fine.

Reply:
Done 👍 
I wasn't able to do that before since `maybeExtractNestedJar` cannot be static